docs/cudf/source/user_guide/cudf.CategoricalDtype.rst

```diff
@@ -0,0 +1,19 @@
+cudf.CategoricalDtype
+=====================
+
+.. currentmodule:: cudf
+
+.. autoclass:: CategoricalDtype
+   :members: categories, ordered, from_pandas, to_pandas
+
+   
+   
+..
+   HACK -- the point here is that we don't want this to appear in the output, but the autosummary should still generate the pages.
+   .. autosummary::
+      :toctree:
+      
+      CategoricalDtype.categories
+      CategoricalDtype.ordered
+      CategoricalDtype.from_pandas
+      CategoricalDtype.to_pandas
```

Comment:
Maybe I misunderstood how this hack works. I thought that we were literally just commenting out the default autosummary behavior so that the attributes wouldn't show up. Isn't all of the content here just a comment, or is it somehow injected into the contents of the autosummary generated by the autoclass?

Reply:
So yes, mentioning just the following:
```
.. autoclass:: CategoricalDtype
   :members: categories, ordered, from_pandas, to_pandas
``` 
will not be generating the respective content and pages for `categories`, `ordered`, `from_pandas` and `to_pandas`...
But, if we remove specifying `:members:` sphinx just lists all the inhertied methods and attributes without any dedicated pages or hyperlinks :(